CRUD_operations.c

```diff
@@ -0,0 +1,131 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include<stdbool.h>
+typedef struct USER {
+    int id;
+    char name[50];
+    int age;
+} User;
+void createUser() {
+    User user;
+    FILE *file = fopen("user.txt", "a");
+    if (!file) {
+        printf("Error opening file\n");
+        return;
+    }
+    printf("Enter User ID: ");
+    scanf("%d", &user.id);

```

Comment:
can we handle read input and printing output in separate function? take care at all places.